monai/bundle/utils.py

```diff
@@ -9,10 +9,200 @@
+
+
+DEFAULT_METADATA = """
+{{
+    "version": "0.0.1",
+    "changelog": {{
+        "0.0.1": "Initial version",
+    }},
+    "monai_version": "{MONAI}",
+    "pytorch_version": "{Pytorch}",
+    "numpy_version": "{Numpy}",
+    "optional_packages_version": {{}},
+    "task": "Describe what the network predicts",
+    "description": "A longer description of what the network does, use context, inputs, outputs, etc.",
+    "authors": "Your Name Here",
+    "copyright": "Copyright (c) Your Name Here",
+    "network_data_format":{{
+        "inputs": {{
+        }},
+        "outputs":{{
+        }}
+    }}
+}}
+""".format(
+    **get_config_values()
+)
+
+DEFAULT_INFERENCE = """
+{
+    "imports": ["$import glob"],
+    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "ckpt_path": "$@bundle_root + '/models/model.pt'",
+    "dataset_dir": "/workspace/data",
+    "datalist": "$list(sorted(glob.glob(@dataset_dir + '/*.jpeg')))",
+    "network_def": {
+        "_target_": "???",
+        "spatial_dims": 2,
+    },
+    "network": "$@network_def.to(@device)",
+    "preprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "LoadImaged",
+                "keys": "image"
+            },
+            {
+                "_target_": "AddChanneld",
+                "keys": "image"
+            },
+            {
+                "_target_": "ScaleIntensityd",
+                "keys": "image"
+            },
+            {
+                "_target_": "EnsureTyped",
+                "keys": "image",
+                "device": "@device"
+            }
+        ]
+    },
+    "dataset": {
+        "_target_": "Dataset",
+        "data": "$[{'image': i} for i in @datalist]",
+        "transform": "@preprocessing"
+    },
+    "dataloader": {
+        "_target_": "DataLoader",
+        "dataset": "@dataset",
+        "batch_size": 1,
+        "shuffle": false,
+        "num_workers": 0
+    },
+    "inferer": {
+        "_target_": "SimpleInferer"
+    },
+    "postprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "Activationsd",
+                "keys": "pred",
+                "softmax": true
+            },
+            {
+                "_target_": "AsDiscreted",
+                "keys": "pred",
+                "argmax": true
+            }
+        ]
+    },
+    "handlers": [
+        {
+            "_target_": "CheckpointLoader",
+            "_disabled_": "$not os.path.exists(@ckpt_path)",
+            "load_path": "@ckpt_path",
+            "load_dict": {"model": "@network"}
+        }
+    ],
+    "evaluator": {
+        "_target_": "SupervisedEvaluator",
+        "device": "@device",
+        "val_data_loader": "@dataloader",
+        "network": "@network",
+        "inferer": "@inferer",
+        "postprocessing": "@postprocessing",
+        "val_handlers": "@handlers"
+    }
```

Comment:
Please add the `evaluating` section according to our latest design:
https://github.com/Project-MONAI/tutorials/blob/main/modules/bundle/spleen_segmentation/configs/inference.json#L128

Thanks.